### PR TITLE
Added ShadPS4 Flatpak

### PR DESCRIPTION
**Pull request by Aeonitis**
Addressing issue #1309, this PR is a draft I created in my best ability to help push the [ShadPS4 Flatpak](https://github.com/shadps4-emu/shadPS4-flatpak) for inclusion in the Emudeck.

Please feel free to criticize this PR as I am new to this project. I have the flatpak install function as a TODO in the comments.

┆Issue is synchronized with this [Trello card](https://trello.com/c/7igLxQ33) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by Aeonitis** [comment]
**To be confirmed**
---
- In [userConfigurations.json](https://github.com/dragoonDorise/EmuDeck/pull/1357/commits/2bbc899c277ec7d285feeb43e810e4a603dafd58#diff-23fd08a7288fbf5cfb6712165429499151bd2017dce7c98a7353d7f530fd1cd4) I need to find out what the id should be for this emulator, I am unsure where it's fetched from, probably an internal db?
- I believe ShadPS4 uses a Playstation profile, not SRM as of now...
https://github.com/dragoonDorise/EmuDeck/blob/84aee7c3e7cb950cc6b39d1a93cc868344e0d4ed/README.md?plain=1#L20)

File: configs/steam-deck-romate/userConfigurations.json [hunk 1]
```diff
@@ -1203,6 +1203,22 @@
       "category":"PlayStation 3",
       "name":"PlayStation 3"
    },
+   {
+      "id": "12345678901234567",
```
Comment (Aeonitis) [inline configs/steam-deck-romate/userConfigurations.json:1207]: To be confirmed with repo mods/admins

Comment (Godsbane) [reply configs/steam-deck-romate/userConfigurations.json:1207]: @brenoprata10 made rommate. I forget how to update it, as i haven't touched this in a while. Maybe he can chime in.

Comment (brenoprata10) [reply configs/steam-deck-romate/userConfigurations.json:1207]: All good 👍

Comment (Aeonitis) [reply configs/steam-deck-romate/userConfigurations.json:1207]: Can you explain how the id works?

Is it supposed to be a secret?

Comment (brenoprata10) [reply configs/steam-deck-romate/userConfigurations.json:1207]: The id is not being used

Comment (Aeonitis) [reply configs/steam-deck-romate/userConfigurations.json:1207]: Ok, so what I understood from what you said is that it is just a placeholder, great!

Thank you

**Comment by dragoonDorise** [comment]
@Aeonitis is the flatpack official from the ShadPS4 devs?

**Comment by Aeonitis** [comment]
> @Aeonitis is the flatpack official from the ShadPS4 devs?

Yes.

- [Official Site](https://shadps4.net)
- [Official Forums](https://forums.shadps4.net)
- [SubReddit](https://www.reddit.com/r/shadps4)
- Flatpak: https://github.com/shadps4-emu/shadPS4-flatpak
- Emulator Source: https://github.com/shadps4-emu/shadPS4
- Compatibility List: https://github.com/shadps4-emu/shadps4-game-compatibility
- Github User: https://github.com/shadps4-emu
- Coming soon (Binaries repos): [Linux](https://github.com/shadps4-emu/shadps4-binaries-Linux), [Mac](https://github.com/shadps4-emu/shadps4-binaries-Mac), [Windows](https://github.com/shadps4-emu/shadps4-binaries-Windows)

File: versions.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
   "rpcs3": { "id": "rpcs3", "code": "RPCS3", "version": 0 },
   "yuzu": { "id": "yuzu", "code": "Yuzu", "version": 1 },
   "ryujinx": { "id": "ryujinx", "code": "Ryujinx", "version": 4 },
+  "shadps4": { "id": "shadps4", "code": "ShadPS4", "version": 0 },
```
Comment (Aeonitis) [inline versions.json:14]: Unsure if exact semantic version (currently `0.4.0`) will be required here yet...

**Pull request merged**